### PR TITLE
Update NodeJS SDK docs for 1.4.0

### PR DESCRIPTION
**Pull request by jonathannorris**

### CODE REVIEW

File: docs/sdk/server-side-sdks/node.md [hunk 1]
```diff
@@ -113,11 +125,36 @@ See [getVariables](https://docs.devcycle.com/bucketing-api/#operation/getVariabl
...
+    target: 'new_subscription',
+    value: 100.1,
+    date: Date.now()
+}
+dvcClient.track(user, event)
+```
+
+### Flush Events
+
+If you would like to force a flush of events in the event queue, you can call `flushEvents()`.
```
Comment (andrewtraviss) [inline docs/sdk/server-side-sdks/node.md:155]: This section should mention that an error will result if flushEvents() is called when there is a flush in progress.

Also, shouldn't we expose a function to check if a flush is in progress, if we expose the ability to manually flush?

**Review by andrewtraviss — CHANGES_REQUESTED** [review]
See inline comments

File: docs/sdk/server-side-sdks/node.md (hunk 1 above)
Comment (ajwootto) [reply docs/sdk/server-side-sdks/node.md:155]: the user-facing flush method should not cause an error to happen if a flush is in progress.
The only thing it should do is make sure that if the current flush in progress finishes, and events are in the queue, another flush will happen immediately.

Comment (jonathannorris) [reply docs/sdk/server-side-sdks/node.md:155]: Currently, it just logs an error if it receives an error from the WASM: 
image

**Review by VicV — APPROVED** [review]

**Pull request merged**